File: iniciante/javascript/1159.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

function somaParesConsecutivos(a, b = 5){
    let soma = 0
    let count = 0
    for(let i = a;count < b; i++ ){
        if(i % 2 === 0){
            soma += i
            count++
        }
    }
    return soma
}

do{
    var entrada = parseInt(lines.shift())
    if(entrada !== 0)
        console.log(somaParesConsecutivos(entrada))
}while(entrada != 0)

File: iniciante/python/1159.py
# -*- coding: utf-8 -*-

soma = 0
count_par = 0
while True:
    entrada = int(input())
    if entrada == 0:
        break
    while True:
        if count_par == 5:
            break
        if entrada % 2 == 0:
            soma += entrada
            count_par += 1
        entrada += 1
    print(soma)
    soma = 0
    count_par = 0
